P5/Source/Specs/att.breaking.xml

```diff
@@ -46,7 +46,7 @@ of any adjacent whitespace</desc>
       </valList>
       <exemplum xml:lang="en">
         <p>In the following lines from the <title level="a">Dream of the Rood</title>, 
-      linebreaks occur in the middle of the words <mentioned>lāðost</mentioned> 
+      line beginnings occur in the middle of the words <mentioned>lāðost</mentioned> 
```

Comment:
I think something like
~~~xml
    <p>In the following lines from the <title level="a">Dream of the Rood</title>, 
      the words <mentioned>lāðost</mentioned> and <mentioned>reord-berendum</mentioned>
      each start on one line and continue on the next.</p>
~~~
would be better.